workflows/dataflow/setup.py

```diff
@@ -3,13 +3,14 @@
 dependencies = [
     "dask==2.13.0",
     "f90nml==1.1.2",
+    "fsspec==0.7.1",
+    "gcsfs==0.6.1",
```

Comment:
are these the versions we want to pin here?

Reply:
Good catch. I just reverted these to the old values. I will fix.